test/test_transforms.py

```diff
@@ -462,6 +462,15 @@ def test_resize_size_equals_small_edge_size(height, width):
     result = t(img)
     assert max(result.size) == max_size
 
+def test_resize_equal_input_output_sizes(height, width):
+    # Regression test for https://github.com/pytorch/vision/issues/7518
+    height, width = 28, 27
+    img = Image.new("RGB", size=(width, height))
+
+    t = transforms.Resize(((height, width)), antialias=True)
```

Comment:
Why `size` arg here is a tuple of tuple ?

Reply:
Just part of my sloppy initial commit...